usr/share/steamos-compositor-plus/bin/dpkg-query

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
But wouldn't the steam client call the original `dpkg-query` and not this script?

Reply:
For this I added a path export to the steamos-session script which put our bin directory first. This ensures that steam always execute our script. I added some safeguards into it in case Steam wants to use dpkg-query for anything else than getting the steamos-updatelevel version, then it will redirect to the original dpkg-query if it exists.

Reply:
I see, nice approach.

Reply:
After some further testing, I concluded that the while loop in this script was causing some issues when a different command was used. I've pushed a fix for it into this pull request.